cypress/support/step_definition/tasks/createAndDeleteTasks.spec.js

```diff
@@ -0,0 +1,48 @@
+import Task from '../../pages/tasks/pageTasks'
+
+const task = new Task()
+
+//Scenario: Teacher creates task as draft from room
+//Given ('I am logged in as a 'teacher' at 'brb'')
+//step defined -->\step_definition\authentication\loginStep.spec.js
+
+//When I go to rooms overview
+//step defined -->\step_definition\course\commonCourseSteps.spec.js
+
+//And I go to room 'Course with subject and tasks'
+//step defined -->\step_definition\course\commonCourseSteps.spec.js
+
+//And('I click on FAB to create task')
+//step defined -->\step_definition\course\commonCourseSteps.spec.js
+
+Then('I can see create task page', () => {
+  task.seeCreateTaskPage()
+})
+
+// And('file upload button is disabled', () => {
+//   task.checkFileUploadButtonIsDisabled()
+// })
+
+When ('I enter title {string}', (taskTitle) => {
+  task.enterTaskTitle(taskTitle)
+})
+
+And ('I click on Enable Group Submission', () => {
+  task.clickOnGroupSubmissionCheckbox()
+})
+
+And ('I click on Draft', () => {
+  task.clickOnDraftCheckbox()
+})
+
+And ('I set task-visibility-start-date to {string} at {string}', (visibilityStartDate, visibilityStartTime) => {
+  task.setVisibilityStartDate(visibilityStartDate, visibilityStartTime)
+})
+
+And ('I set task-visibility-due-date to {string} at {string}', (visibilityDueDate, visibilityDueTime) => {
+  task.setVisibilityDueDate(visibilityDueDate, visibilityDueTime)
+})
+
+And ('I enter task description {string}', (taskDescription) => {
+  task.setTaskDescription(taskDescription)
+})
```

Comment:
further following steps as in the feature file are also defined somewhere else that we can put location comments here too:
-> And I click on button Submit
-> Then I can see room page 'Course with subject and tasks'
-> And I can see task 'Task Creation and Deletion Test'

also for second scenario (delete task) all steps are defined somewhere so location info can be commented out here too in this step definition file
-> Scenario: Teacher deletes task from room
    -> Given I am logged in as a 'teacher1' at 'brb'
    ->When I go to rooms overview
    ->And I go to room 'Course with subject and tasks'
    ->When I click on dot menu of content 'Task Creation and Deletion Test'
    ->And I click on Delete in dot menu
    ->And I click on Cancel in confirmation window
    ->Then I can see task 'Task Creation and Deletion Test'
    ->When I click on dot menu of content 'Task Creation and Deletion Test'
      ->And I click on Delete in dot menu
     -> And I click on Delete in confirmation window
     ->Then I can see room page 'Course with subject and tasks'
     ->And I can not see task 'Task Creation and Deletion Test'

Reply:
Yes, I forgot to write the steps that are defined somewhere else.
I'll write and comment them out.